src/common/modals/compositions/DepositModal/SwapStep.vue

```diff
@@ -0,0 +1,207 @@
+<template>
+  <div class="swap-step">
+    <input-field
+      v-model="form.amount"
+      :placeholder="$t(`swap-step.amount-placeholder`)"
+      :error-message="getFieldErrorMessage('amount')"
+      :disabled="isLoadFailed || !isLoaded"
+      @blur="touchField('amount')"
+    />
+    <div class="swap-step__desc-wrp">
+      <div class="swap-step__desc">
+        <span class="swap-step__desc-text">
+          {{ $t('swap-step.eth-amount-text') }}
+        </span>
+        <span class="swap-step__desc-amount">
+          {{ humanizedEstimations.estimatedTokenOutAmount }}
+        </span>
+      </div>
+      <div class="swap-step__desc">
+        <span class="swap-step__desc-text">
+          {{ $t('swap-step.estimated-fee-text') }}
+        </span>
+        <span class="swap-step__desc-amount">
+          {{ humanizedEstimations.estimatedGasCost }}
+        </span>
+      </div>
+    </div>
+    <div class="swap-step__buttons">
+      <app-button
+        class="swap-step__button"
+        color="secondary"
+        :text="$t('swap-step.cancel-btn')"
+        @click="emit('cancel')"
+      />
+      <app-button
+        class="swap-step__button"
+        :text="mainButtonText"
+        :disabled="!isFieldsValid || !isLoaded"
+        @click="submit"
+      />
+    </div>
+  </div>
```

Comment:
If something looks like form and behaves like form - it should be form.

- Replace `<div>` with `<form>` component
- Move `submit` handler from `@click` button to the form `@submit` event

Reply:
done